bin/cloud-login

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+if [ -n "${CLOUD_LOGGED_IN+1}" ]
+then
+  echo "Already logged in to gloud provider"
+  return 0
+fi
+
+if [ ! -z "${AWS_ACCESS_KEY_ID}" ]; then
+    prepare-awscli
+    CLOUD_LOGGED_IN=true
+    return 0
+fi
+
+if [ ! -z "${GCLOUD_KEY}" ]; then
+    prepare-gcloud
+    CLOUD_LOGGED_IN=true
+    return 0
+fi
+
+echo "Something unexpected occured authenticating to cloud provider. Attempting to proceed..." 
```

Comment:
This line may need a similar setup to the `CLOUD_LOGGED_IN` var. If rok8s-scripts is used but doesn't need AWS or GCP this line would wind up being printed each time one of our scripts calls this.

Reply:
I don't have a good way around that except to remove it.